python/erdos/streams.py

```diff
@@ -403,7 +408,7 @@ def send(self, msg: Message):
         self._internal_stream.send(internal_msg)
 
 
-class ExtractStream:
+class ExtractStream(Generic[T]):
```

Comment:
I believe `ExtractStream` is also of type `Stream[T]`.

Reply:
Great thanks will fix this and push shortly